src/Apps/Order/Routes/Shipping2/FulfillmentDetails.tsx

```diff
@@ -99,12 +241,13 @@ export const FulfillmentDetails: FC<FulfillmentDetailsProps> = props => {
       verifyAddressNow={verifyAddressNow}
       onSubmit={handleSubmit}
       availableFulfillmentTypes={availableFulfillmentTypes}
+      forceNewAddressFormMode={forceNewAddressFormMode}
     />
   )
 }
 
 export const FulfillmentDetailsFragmentContainer = createFragmentContainer(
-  FulfillmentDetails,
+  injectDialog(FulfillmentDetails),
```

Comment:
🔴 oops, I don't think this is needed any more. Dialog is moved into context helpers.